src/lib/widgets/audio/ui/audio.svelte

```diff
@@ -0,0 +1,42 @@
+<script lang="ts">
+  import { audioStore } from '$lib/features/fetch-audio';
+  import ScrubbingBar from './scrubbing-bar.svelte';
+
+  let audioEl = null as HTMLAudioElement | null;
```

Comment:
I think you can leave it as 
```suggestion
  let audioEl: HTMLAudioElement;
```
to avoid the checks in the beginning of every function, since it will always be available whenever any of these functions is called. The only problem can happen when it used outside any function, but that should be clear straight away